### PR TITLE
Use commonmarker instead of github-markdown

### LINKED ISSUE #76: use commonmarker instead of github-markdown
https://github.com/gjtorikian/commonmarker
https://github.com/jch/html-pipeline/issues/246#issuecomment-280216711

### PR DESCRIPTION
**Pull request by nikolas**
Closes #76

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (nikolas) [inline .travis.yml]: I updated to ubuntu 14.04 here to resolve a problem compiling native extensions for commonmarker. See: https://travis-ci.org/mcfiredrill/forttree/builds/235902726#L355

**Comment by nikolas** [comment]
it's a dependency of the new release of html-pipeline: https://github.com/jch/html-pipeline/pull/274

**Comment by mcfiredrill** [comment]
Oh you are right it just needs to be in the Gemfile. This needs a rebase though.

**Comment by nikolas** [comment]
I like rebases too better than merges, but i'm kind of drunk i resolved the conflict in the github interface. damn, now something's wrong. I'll rebase it later unless you want to fix it up.

**Comment by mcfiredrill** [comment]
ok I'll merge it and re-run bundle install and try to fix it

**Pull request merged**